Subject: winbind plugin: Add -DMPPE=1 to eliminate compiler warnings
Origin: upstream, commit:2b05e22c62095e97dd0a97e4b5588402c2185071
From: Paul Mackerras <paulus@samba.org>

When compiling the winbind plugin, we need an equivalent definition
of the MPPE symbol to that which applied when the main pppd was
compiled.  This adds that to Makefile.linux.

Reported-by: Mike Gilbert <floppym@gentoo.org>
Signed-off-by: Paul Mackerras <paulus@samba.org>

diff --git a/pppd/plugins/Makefile.linux b/pppd/plugins/Makefile.linux
index 0a7ec7badb2a..ab8cf50d9472 100644
--- a/pppd/plugins/Makefile.linux
+++ b/pppd/plugins/Makefile.linux
@@ -14,6 +14,13 @@ SUBDIRS := rp-pppoe pppoatm pppol2tp
 SUBDIRS += radius
 PLUGINS := minconn.so passprompt.so passwordfd.so winbind.so
 
+# This setting should match the one in ../Makefile.linux
+MPPE=y
+
+ifdef MPPE
+CFLAGS   += -DMPPE=1
+endif
+
 # include dependencies if present
 ifeq (.depend,$(wildcard .depend))
 include .depend
-- 
2.1.0

